tests/installation/first_boot.pm

```diff
@@ -94,6 +94,14 @@ sub run {
         send_key "esc";
         assert_screen 'generic-desktop';
     }
+    # boo#1105302 - Disable kernel watchdog on aarch64
+    if (check_var('ARCH', 'aarch64')) {
```

Comment:
Do we have this bug on SLE? I believe we need to limit this.
In general I would say that it belongs to other test module, e.g. `system_prepare`, but I also do understand a need to perform workaround as early as possible. Potentially, we could have introduced a new test-module and schedule it only when required. 
Also, take a look [here](https://github.com/os-autoinst/openQA/blob/master/docs/WritingTests.asciidoc#capturing-kernel-exceptions-and-or-any-other-exceptions-from-the-serial-console) it may be useful for the bug detection.

Reply:
It could make sence to change this permanently using something like this:

```bash
echo "kernel.watchdog_thresh = 0" > /etc/sysctl.d/watchdog.conf
```

Reply:
@rwx788 I do know if there is this bug on SLE. I think it depends if the machine is very powerful and it also depends on its load at that time. 
So,  I think SLE is also affected.

Reply:
And what's about this proposal https://github.com/os-autoinst/os-autoinst/pull/1016 ? It didn't help, or we didn't deploy to o3?

Reply:
@asdil12 let's add it to sysctl conf if needed later. I would prefer to minimize modifications of tested system.

Reply:
> And what's about this proposal [os-autoinst/os-autoinst#1016](https://github.com/os-autoinst/os-autoinst/pull/1016) ? It didn't help, or we didn't deploy to o3?

@rwx788 I think @dirkmueller did deploy a workaround, but it still takes time.

Reply:
As @rwx788 stated I am also fine with the workaround but please move to the test module `system_prepare`

Reply:
> As @rwx788 stated I am also fine with the workaround but please move to the test module `system_prepare`

The problem is system_prepare is called too late. See: https://openqa.opensuse.org/tests/761774#step/hostname/3